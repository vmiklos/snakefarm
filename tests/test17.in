out 1
time 1
load test17.map
addplayer 1
addsnake 1 1 0 1 1 0
11 1
10 1
9 1
endsnake
addplayer 2
addsnake 2 2 0 1 1 0
18 0
17 1
16 1
endsnake
addplayer 3
addsnake 3 3 0 1 1 0
25 0
24 1
23 1
endsnake
comment # mindegyik kigyo 3 hosszu
comment # az elsoben 3 ko van
showsnake 1
comment # a masodikban es a harmadikban 2
showsnake 2
showsnake 3
comment # ket nyertesnek kell lennie, a masodiknak es a harmadiknak
step
exit