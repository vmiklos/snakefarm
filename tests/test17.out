Event MapLoad test17.map
# mindegyik kigyo 3 hosszu
# az elsoben 3 ko van
Snake 1
Prop 0 1 1
1 11 1
2 10 1
3 9 1
endsnake
# a masodikban es a harmadikban 2
Snake 2
Prop 0 1 1
1 18 0
2 17 1
3 16 1
endsnake
Snake 3
Prop 0 1 1
1 25 0
2 24 1
3 23 1
endsnake
# ket nyertesnek kell lennie, a masodiknak es a harmadiknak
Event Step 1
End Step
Event Step 2
End Step
Event Step 3
End Step
Event Win 2
Event Win 3
Event Exit